docs/ECLStandardLibraryReference/SLR-Mods/MonitorLogicalFileName.xml

```diff
@@ -98,6 +97,10 @@
   <emphasis>event</emphasis> with the name of the triggering object as the
   event subtype (see the EVENT function).</para>
 
+  <para>This function does not support wildcard characters. To monitor for
+  files using wildcards, use the <link
```

Comment:
@JamesDeFabia 
Is it just me, or would that sound better without the 'for' ?
i.e. "To monitor files using wildcards .. "